<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotPhaseStream</class>
 <widget class="QWidget" name="PlotPhaseStream">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1244</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GraphicsLayoutWidget" name="graphicsLayoutWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="callGetPhaseStreamTime">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>callGetPhaseStreamTime?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="nMaxToPlot">
        <property name="toolTip">
         <string>maximum number of points to plot</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>1e3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1e4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1e5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1e6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1e7</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="downsamplingFactor">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>DownSampling factor </string>
        </property>
        <property name="text">
         <string>????</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="symbolSize">
        <property name="toolTip">
         <string>Symbol Size</string>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="whatToPlot">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>phases</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="duration">
        <property name="maximum">
         <double>2.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stepLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="streamState">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>streamState?</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="streamProgressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="datetimeClock">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>datetimeClock?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="loFreqHz">
        <property name="toolTip">
         <string>LO freq (Hz)</string>
        </property>
        <property name="text">
         <string>5.1e9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fToneMin">
        <property name="toolTip">
         <string>f tone (Hz)</string>
        </property>
        <property name="text">
         <string>5.7e9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fToneMax">
        <property name="toolTip">
         <string>f tone (Hz)</string>
        </property>
        <property name="text">
         <string>6.0e9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="nTone">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1024</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generateToneState">
        <property name="text">
         <string>generateToneState?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="toneGenerationProgressBar">
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="iFreq"/>
      </item>
      <item>
       <widget class="QPushButton" name="stop">
        <property name="cursor">
         <cursorShape>ClosedHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
